MyOnboarding/src/MyOnboardingApp.Api/Controllers/TodoListController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.Web.Http;
+using MyOnboardingApp.Api.Models;
+
+namespace MyOnboardingApp.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [RoutePrefix("api/v{version:apiVersion}/todolist")]
+    [Route("")]
+    public class TodoListController : ApiController
+    {
+        public static TodoListItem DefaultItem =
```

Comment:
Use different Id for dummy purposes, something like 'dummyId' or 'dog' or anything easy.